requirements.txt

```diff
@@ -1,5 +1,6 @@
 attrs==17.4.0
 certifi==2018.4.16
+click==8.1.3
```

Comment:
You have to also add this to setup.py as install_requires as such:

```
    install_requires = [
        'click >= 8.1,< 9'
    ],
```

Ideally right the license param would be good. And while you are at it, can you add pygtrie to this which has been an issue for a while. The version string should be 'pygtrie >= 2.2, <3'.

Reply:
Why do you want to add this requirement to `install_requires`, while the other dependencies are not listed here?

Reply:
Ok, spoke too soon. I see you're only actually using pygtrie, and now click as well. I'll add them both!